backend/scripts/demo_topics.sh

```
@@ -0,0 +1,69 @@
+set -euo pipefail
+cd "$(dirname "$0")/.."
+
+BASE_URL="http://localhost:8000"
+DB_CONTAINER="aurora-db"
+
+echo "[1/8] Ensuring API server is running..."
+if ! curl -sSf "${BASE_URL}" >/dev/null 2>&1; then
+  echo "Server not responding on ${BASE_URL}. Starting server..."
+  nohup npm run start:prod >/tmp/aurora_server.log 2>&1 &
+  echo "Waiting for server to become ready..."
+  for i in {1..30}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)